game/src/main/scala/io/aigar/game/Cell.scala

```diff
@@ -46,6 +47,14 @@ class Cell(id: Int, startPosition: Vector2 = new Vector2(0f, 0f)) {
     return if (dir.magnitude > 0) dir.normalize * value else new Vector2(0f,0f)
   }
 
+  def contains(pos: Vector2): Boolean = {
+    if ( mass >= sqrt(pow(position.x - pos.x,2) + pow(position.y - pos.y,2))){
```

Comment:
The math library that we are using has a [`distanceTo`](https://github.com/jpbetz/subspace/blob/master/subspace/src/main/scala/com/github/jpbetz/subspace/Vector2.scala#L79) function. Could we use it? :) I think it would look a bit cleaner to have something like `if (position.distanceTo(pos) <= mass)` or similar.

And, actually, we could just return the boolean result itself instead of doing `if(bool) true else false`: `return position.distanceTo(pos) <= mass`

What do you think?